src/entities/Place/model.ts

```diff
@@ -354,8 +354,8 @@ export default class PlaceModel extends Model<PlaceAttributes> {
       hotScenePlaces.reverse()
     }
 
-    const from = numeric(offset || 0, { min: 0 })
-    const to = numeric(from + (limit || 100), { min: 0, max: 100 })
+    const from = numeric(offset || 0, { min: 0 }) ?? 0
+    const to = numeric(from + (limit || 100), { min: 0, max: 100 }) ?? 100
```

Comment:
do we need this? we are already sending 0 or 100 to the function

Reply:
The function returns `number | null`, making us provide a default number.